src/test/codegen/sanitizer_scs_attr_check.rs

```diff
@@ -0,0 +1,12 @@
+// This tests that the shadowcallstack attribute is
+// applied when enabling the shadow-call-stack sanitizer.
+//
+// needs-sanitizer-shadow-call-stack
+// compile-flags: -Zsanitizer=shadow-call-stack
+
+#![crate_type = "lib"]
+
+// CHECK: ; Function Attrs:{{.*}}shadowcallstack
+pub fn scs() {}
+
+// CHECK: attributes #0 = {{.*}}shadowcallstack
```

Comment:
This is the attribute set associated with the `scs` definition, right? What’s the point of checking this the second time?

Reply:
I noticed it was emitted here so I figured I should check it, but you're right -- the important bit here is that shadowcallstack is associated with the scs function. I've removed this since it's unnecessary.